### DISCUSSION TITLE
Login as a user in new incognito window

### DISCUSSION THREAD

**Discussion by tejastg007** [tprouvot/Salesforce-Inspector-reloaded#272] [Ideas]
The option login as a user logs in the same window, overriding the current user. Instead, if the new user logs in the incognito window, that would be great so that you won't need to login back to the current admin user.

**Comment by tprouvot** [comment]
Hi @tejastg007 
The actual authentication relies on the sid cookie set by Salesforce.
Opening the window in a private one would prevent us to use the sid, I'm not sure this would be possible without an important rework.

**Reply by tejastg007** [reply]
Oh, i don't have thorough knowledge about it, but I saw this feature in the Salesforce organizer extension. There you can log in as any user in the current window as well as the incognito window. So I thought it would be great if reloaded also includes it. Thank you

**Comment by tprouvot** [comment]
Hi @tejastg007,

I was wrong when I told you it was complex to implement ! The functionality is coming 🚀

**Reply by tejastg007** [reply]
great! Thank you for considering the feature request :)

**Comment by tprouvot** [comment]
@tejastg007 Available in the beta version !
